.github/CONTRIBUTING.md

```diff
@@ -1,3 +1,50 @@
+# Contributing
+
+Thank you for contributing to web3.js! We appreciate your interest and welcome any contributions that can help improve our documentation, functionality, addressing bugs, or any other aspects that can contribute to the overall improvement of our project. Before you start contributing, please take a moment to review the guidelines below.
+
+## Help and Support
+
+If you face any issues while contributing, or you want any type of support, we encourage you to join our [Discord Community](https://discord.com/invite/3shNX8cqGR) and ask any question in the `#web3js-general` channel, and/or submit a [New Issue](https://github.com/web3/web3.js/issues/new).
+
+## Prerequisites
+
+- [NodeJS](https://nodejs.org/) (LTS)
+- [Yarn](https://yarnpkg.com/)
+
+## Contributing to the docs
+
+1. **Fork the docs:** Start by forking our repository to your GitHub account.
+
+2. **Clone the repo:** Clone the forked repository to your local machine using the following command
+```bash
+git clone https://github.com/your-username/web3.js.git
+```
+3. **Create a Branch:** Create a new branch for your changes with a descriptive name.
+    **NOTE: The branch name must include the issue number (if there is no issue created for your contribution, please create one)**
+```bash
+git checkout -b issue-name-1234
+```
+4. **Navigate to the docs folder:** `cd web3.js/docs/docs`
+
+5. **Install docusaurus:** 
+```bash
+yarn add docusaurus
```

Comment:
Its already added so only `yarn` in doc path should be enough and it will installed dependencies listed in doc package.json